source/release-notes/2.3.0.rst

```diff
@@ -0,0 +1,51 @@
+.. _2.3.0:
+
+===========================================
+Percona Monitoring and Management 2.3.0
+===========================================
+
+:Date: February 17, 2020
+
+`Percona Monitoring and Management <https://www.percona.com/doc/percona-monitoring-and-management/2.x/index.html>`_ (PMM) is a free and open-source platform for managing and monitoring |MySQL|, |MongoDB|, and |postgresql| performance.
+
+For *PMM* install instructions, see :ref:`pmm.deploying.server` and :ref:`pmm.deploying.client`.
+
+.. note:: PMM 2 is designed to be used as a new installation — your existing
+   PMM 1 environment can't be upgraded to this version.
+
+Improvements and new features
+=============================
+
+* :pmmbug:`5064` and :pmmbug:`5065`: Starting from this release, users will be
+  able to integrate PMM with an external Alertmanager, configuring this via
+  the PMM Settings dashboard, to pass alert rules to PMM
```

Comment:
Can we clarify there: " ... to integrate PMM with an external Alertmanager by specifying external Alertmanager URL and Alert Rules to be execured iinside PMM server" . (something like this will be better to let people know - we let you spesify  2 parameters  only)